packages/kotti-button/src/button.vue

```diff
@@ -103,3 +107,111 @@ export default {
 	},
 }
 </script>
+<style lang="scss">
+$default-button-height: 32px;
+$large-button-height: 40px;
+$small-button-height: 24px;
```

Comment:
maybe good to semantically create a difference between these local variables and the globals?

Reply:
I'm not sure to understand what you mean there, do you have an example ?

Reply:
these variables should probably be derived from existing $unit measurements already used instead of being hard code declared